samples/MicroServices/NBB.Contracts/NBB.Contracts.Application/DomainEventHandlers/ReadModelGenerator.cs

```diff
@@ -21,62 +21,59 @@ public ReadModelGenerator(ICrudRepository<ContractReadModel> contractReadModelRe
             _contractReadModelRepository = contractReadModelRepository;
         }
 
-        public async Task Handle(ContractCreated @event, CancellationToken cancellationToken)
+        public async Task Handle(ContractCreated notification, CancellationToken cancellationToken)
```

Comment:
It seams a little odd to name events as "notification" even if they are in the implementation of INotificationHandler